src/scss/functions/_output.scss

```diff
@@ -29,12 +29,20 @@
 	/*
 		The overruler to create all the css
 	*/
-	@if $output-full == true {
+	@if map-get($output, 'full') == true {
 		$doOutput: true;
 	}
 
 	/*
-        In the end, determine if this prop should return anything.
-    */
+      In the end, determine if this prop should return anything.
+  */
 	@return $doOutput;
 }
+
+/*
+  Allow to create custom configuration with $henris
+  Check if the henris is defined to override $ouput
+ */
+@if(variable_exists(henris)){
```

Comment:
It's to have a distinct custom variable that can be merged with `$ouput` to change configuration. Could be probably a shorter way to do this, but I like the idea to have an $henris var.

In my project I got something like this to start with $henris
```
$henris: (
 'full': false,
 'grid-cssgrid': true,
 'button': false,
);


@import 'henris';
@import './scss/index';
```

Reply:
I don't think I have any problems with this. Lets just alter the comments to // instead of /* (/* will be outputted in css). 

Maybe change the description to something like:

//
// Allow to use more clear 'henris' approach variable by using $henris instead of $output and
// keep variables in a list in the project to show clearly what the settings are meant for. 
// the $henris variable extends or overrides certain settings in the main output setting. 
// 

@mmeester what do you think?

Reply:
Map has key has been added in favour of the variable exists. Also did some other improvements